openshift/keycloak.app.yaml

```diff
@@ -66,24 +66,24 @@ objects:
           resources: {}
           livenessProbe:
             httpGet:
-              path: /auth/version
+              path: /auth/realms/fabric8
               port: 8080
               scheme: HTTP
             successThreshold: 1
-            failureThreshold: 3
+            failureThreshold: 5
             timeoutSeconds: 10
-            periodSeconds: 10
-            initialDelaySeconds: 60
+            periodSeconds: 5
+            initialDelaySeconds: 120
```

Comment:
@alexeykazakov this should be equal or lower than the readiness probe. You first probe that is alive then that is ready